### ISSUE TITLE
Issue Cleanup Round #1

### ISSUE THREAD

**Issue by f0ssel** [coder/coder#3276]
We have an overwhelming amount of issues that go stale and are better off being remade when they come back up in priority. In our retro today this was brought up again. I'm trying to go through old issues that have the following problems:
- No longer applicable / information is out of date
- Not clearly actionable
- Done / Decided we won't do
- Not detailed enough
- Probably won't prioritize in the next 3 months

Remembering that all issues can easily be reopened at a later date, I would like to close the following:
- https://github.com/coder/coder/issues/44
- https://github.com/coder/coder/issues/358
- https://github.com/coder/coder/issues/359
- https://github.com/coder/coder/issues/470
- https://github.com/coder/coder/issues/479
- https://github.com/coder/coder/issues/519
- https://github.com/coder/coder/issues/581
- https://github.com/coder/coder/issues/589
- https://github.com/coder/coder/issues/603
- https://github.com/coder/coder/issues/611
- https://github.com/coder/coder/issues/613
- https://github.com/coder/coder/issues/614
- https://github.com/coder/coder/issues/619
- https://github.com/coder/coder/issues/620
- https://github.com/coder/coder/issues/665
- https://github.com/coder/coder/issues/672
- https://github.com/coder/coder/issues/686
- https://github.com/coder/coder/issues/695
- https://github.com/coder/coder/issues/722
- https://github.com/coder/coder/issues/726
- https://github.com/coder/coder/issues/736
- https://github.com/coder/coder/issues/764
- https://github.com/coder/coder/issues/778
- https://github.com/coder/coder/issues/790
- https://github.com/coder/coder/issues/791
- https://github.com/coder/coder/issues/792
- https://github.com/coder/coder/issues/793
- https://github.com/coder/coder/issues/798
- https://github.com/coder/coder/issues/842
- https://github.com/coder/coder/issues/843
- https://github.com/coder/coder/issues/905
- https://github.com/coder/coder/issues/912
- https://github.com/coder/coder/issues/945
- https://github.com/coder/coder/issues/979
- https://github.com/coder/coder/issues/990
- https://github.com/coder/coder/issues/997
- https://github.com/coder/coder/issues/1067
- https://github.com/coder/coder/issues/1068
- https://github.com/coder/coder/issues/1078
- https://github.com/coder/coder/issues/1097
- https://github.com/coder/coder/issues/1099
- https://github.com/coder/coder/issues/1224
- https://github.com/coder/coder/issues/1276
- https://github.com/coder/coder/issues/1277
- https://github.com/coder/coder/issues/1286
- https://github.com/coder/coder/issues/1321
- https://github.com/coder/coder/issues/1383
- https://github.com/coder/coder/issues/1391
- https://github.com/coder/coder/issues/1392
- https://github.com/coder/coder/issues/1393
- https://github.com/coder/coder/issues/1401
- https://github.com/coder/coder/issues/1425
- https://github.com/coder/coder/issues/1477
- https://github.com/coder/coder/issues/1479
- https://github.com/coder/coder/issues/1525
- https://github.com/coder/coder/issues/1547
- https://github.com/coder/coder/issues/1565
- https://github.com/coder/coder/issues/1605
- https://github.com/coder/coder/issues/1628
- https://github.com/coder/coder/issues/1631
- https://github.com/coder/coder/issues/1762
- https://github.com/coder/coder/issues/1780

**Comment by kylecarbs** [comment]
I agree with closing all!

**Comment by f0ssel** [comment]
@tjcran @bpmct I would like you both to sign off before the purge

**Comment by bpmct** [comment]
All good to close all except #802 #1554 as I think we'll be doing both relatively soon!

**Comment by presleyp** [comment]
Please keep #1082 #1532 and #961

**Comment by dwahler** [comment]
I think #909, which is currently on the proposed August roadmap, depends on #44. Depending on the implementation, it might also depend on #1391, #1392, #1393, and/or #1605.

Many of these sound like issues that we've said we do plan to implement at some point, even though the issue descriptions might be out of date. Do we plan to reopen them when we eventually get around to them?

**Comment by tjcran** [comment]
> I think #909, which is currently on the proposed August roadmap, depends on #44. Depending on the implementation, it might also depend on #1391, #1392, #1393, and/or #1605.
> 
> Many of these sound like issues that we've said we do plan to implement at some point, even though the issue descriptions might be out of date. Do we plan to reopen them when we eventually get around to them?

@dwahler yes! closing does not mean its forever closed. any of them can always be reopened.

**Comment by f0ssel** [comment]
@dwahler after reading through those tickets it seemed to me the plan and details were out of date. Closing these tickets in no way implies that work won't be planned or worked on, but that these tickets being open and existing are not helping us right now.

**Comment by f0ssel** [comment]
@spikecurtis @dwahler I reopened the ones I think are relevant to your work, sorry for misunderstanding and jumping the gun on that.

**Comment by dwahler** [comment]
@f0ssel No problem at all, and thanks for doing this cleanup!
